web/src/test-utils.tsx

```diff
@@ -6,7 +6,7 @@ import {
 import { HeadManagerContext } from "next/dist/shared/lib/head-manager-context";
 import { RouterContext } from "next/dist/shared/lib/router-context";
 import { NextRouter } from "next/router";
-import { ReactElement, useEffect, PropsWithChildren } from "react";
+import { ReactElement, useEffect, PropsWithChildren, ReactNode } from "react";
```

Comment:
Can't see ReactNode import in use anywhere in this file?

Reply:
Good spot. And `PropsWithChildren` actually. Will sort. You're like a human linter,

Reply:
Pushed a fix